frontend/src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import LoginButton from "../LoginButton/LoginButton";
+import SignupButton from "../SignUpButton/SignupButton";
+import AppLogo from "../AppLogo/AppLogo";
+import HomeButton from "../HomeButton/HomeButton";
+import SavedButton from "../SavedButton/SavedButton";
+import './NavBar.css';
+import ProfileButton from "../ProfileButton/ProfileButton";
+import UserAvatar from "../UserAvatar/UserAvatar";
+
+
+const NavBar = () => {
+
+  return (
+    <nav>
+      <div class="logo">
+        <AppLogo />
+      </div>
+      <div class="navigation">
+        <HomeButton />
+      </div>
+      <div class="navigation">
+        <SavedButton />
+      </div>
+      <div class="navigation">
+        <ProfileButton />
+      </div>
+      <div class="navigation">
+        <UserAvatar />
+        <div>
+          <LoginButton />
+          <SignupButton />
+        </div>
+      </div>
```

Comment:
I don't think that this is quite the way to set up the navBar and here's my thoughts as to why: 
1. Buttons are typically used for actions, like form submissions, so all of these that are set up as button components really should be `<a>` tags.
2. These won't be used in other places, so there really isn't a solid reason that I can come up with for these to be components outside of the navbar, I think they could all just be `<a>` tags inside the navbar component and then styled appropriately. 
3. React router has a really cool way of making `<a>` tags using both the built-in `<Link>` and `<NavLink>` components and I think it would be easier to use those in this instance, both because the routing to different pages behavior is built-in and also because it would avoid prop-drilling excessively between the nav bar and the other button components.
4. I think if we have time, we will want to only allow the user to navigate to some of the pages if they're logged in and that will be even more difficult with these being buttons in separate components, and I think its best to stay away from complexity in this project as much as possible. 

If you have a solid reason for using buttons and pulling them out into separate components, I'm all ears! Otherwise I'd like for this component to use styled-components, import the NavBar component and use that instead of the `<nav>` tag, and then place each link to each page into this component as react-router-dom `<NavLink>` components.  

https://javascript.plainenglish.io/react-router-dom-the-differences-between-navlink-link-and-a-bb5c27a48bfc

I know it's overwhelming to learn new things and try to make the time to keep up with tasks, but this is a learning opportunity so reach out if you have questions or need help getting started